src/core/font_renderer.js

```diff
@@ -122,7 +122,7 @@ var FontRendererFactory = (function FontRendererFactoryClosure() {
   }
 
   function lookupCmap(ranges, unicode) {
-    var code = unicode.charCodeAt(0), gid = 0;
+    var code = unicode.codePointAt(0), gid = 0;
```

Comment:
According to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/codePointAt#Browser_compatibility this (unsurprisingly) isn't supported by IE11.

Please include the necessary `string/code-point-at` polyfill, using [core-js](https://github.com/zloirock/core-js#ecmascript-string-and-regexp), in the `src/shared/compatibility.js` file.